diff --git a/node_modules/@hydrofoil/shaperone-wc-shoelace/elements/sh-sl-autocomplete.js b/node_modules/@hydrofoil/shaperone-wc-shoelace/elements/sh-sl-autocomplete.js
index ef0875a..659f12a 100644
--- a/node_modules/@hydrofoil/shaperone-wc-shoelace/elements/sh-sl-autocomplete.js
+++ b/node_modules/@hydrofoil/shaperone-wc-shoelace/elements/sh-sl-autocomplete.js
@@ -33,6 +33,7 @@ function _arrayWithHoles(arr) { if (Array.isArray(arr)) return arr; }
 import { css, html, LitElement } from 'lit';
 import { customElement, property, query } from 'lit/decorators.js';
 import debounce from 'p-debounce';
+import { rdfs, schema } from '@tpluscode/rdf-ns-builders'
 import { stop } from '../lib/handlers.js';
 import '@shoelace-style/shoelace/dist/components/input/input.js';
 import '@shoelace-style/shoelace/dist/components/icon/icon.js';
@@ -210,7 +211,7 @@ export let ShSlAutocomplete = _decorate([customElement('sh-sl-autocomplete')], f
                         slot="suffix"
                         ?hidden="${!this.selected || !this.clearable}"
                         @click="${this.cleared}"></sl-icon-button>
-        <sl-icon name="${this.loading ? 'arrow-repeat' : 'search'}" slot="suffix"></sl-icon>
+        <sl-icon name="${this.loading ? 'arrow-repeat' : 'search'}" slot="suffix" @click="${this.dispatchSearch}"></sl-icon>
       </sl-input>
       <sl-menu hoist .value=${(_this$selected = this.selected) === null || _this$selected === void 0 ? void 0 : _this$selected.value}
                ?hidden="${this.empty}"
@@ -232,6 +233,11 @@ export let ShSlAutocomplete = _decorate([customElement('sh-sl-autocomplete')], f
             this._menu.show();
           }
         }
+        if (_changedProperties.has('selected')) {
+          this.inputValue = this.selected?.out(schema.name).value
+            || this.selected?.out(rdfs.label).value
+            || ''
+        }
       }
     }, {
       kind: "method",
