diff --git a/node_modules/@types/rdf-loader-code/evalTemplateLiteral.d.ts b/node_modules/@types/rdf-loader-code/evalTemplateLiteral.d.ts
new file mode 100644
index 0000000..a34997c
--- /dev/null
+++ b/node_modules/@types/rdf-loader-code/evalTemplateLiteral.d.ts
@@ -0,0 +1,6 @@
+function evalTemplateLiteral(content: string, options?: {
+  context?: Record<string, unknown>
+  variables?: Map<string, unknown>
+}): string
+
+export = evalTemplateLiteral
