diff --git a/node_modules/hydra-box/lib/middleware/iriTemplate.js b/node_modules/hydra-box/lib/middleware/iriTemplate.js
index b725da2..100702b 100644
--- a/node_modules/hydra-box/lib/middleware/iriTemplate.js
+++ b/node_modules/hydra-box/lib/middleware/iriTemplate.js
@@ -65,13 +65,17 @@ function middleware ({ dataset, term, graph }) {
     const templateParams = $rdf.clownface({ dataset: rdf.dataset() }).blankNode()
 
     Object.entries(req.params).forEach(([key, value]) => {
+      const isQueryParam = key in req.query
       const property = variablePropertyMap.get(key)
 
       if (!property) {
         return
       }
 
-      templateParams.addOut(property, createTermFromVariable({ template: iriTemplateNode, value }))
+      templateParams.addOut(property, createTermFromVariable({
+        template: iriTemplateNode,
+        value: isQueryParam ? value.replace(/\+/g, ' ') : value
+      }))
     })
 
     req.dataset = () => {
